database/migrations/20200225195426_page_component.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex) {
```

Comment:
Thank you for linking many components to each page. So with us saving these components we will need routes to save them into individual slots (an array possibly) so it can be rerender our created store